src/docs/sphinx/blueprint_mesh.rst

```diff
@@ -660,6 +659,30 @@ The Blueprint schema for each entry in the ``nestsets`` section matches the foll
    * nestsets/nestset/windows/window/origin/{i, j, k}
    * nestsets/nestset/windows/window/dims/{i, j, k}
 
+.. note::
+   Many structured AMR codes use global coordinate identifiers when specifying
+   each window's ``origin``. Such coordinates must be transformed to domain-local
+   coordinates to be Blueprint-compliant. Given the global structured origin of
+   a window's associated topology ``topo_origin`` (which isn't in the Blueprint,
+   but is likely stored somewhere in the client code), the global origin can be
+   transformed into a local origin like so:
+
+   .. code:: cpp
+
+       conduit::Node &global_origin = // path to nestset/windows/window/origin
+       conduit::Node &topo_origin = // loaded from client code; {i, j, k} structure
+
+       conduit::NodeIterator origin_it = global_origin.children();
+       while(origin_it.has_next())
+       {
+           conduit::Node &global_dim = origin_it.next();
+           conduit::Node &topo_dim = topo_origin[origin_it.name()];
+
+           conduit::int64 new_dim_val = global_dim.to_int64() - topo_dim.to_int64();
+           conduit::Node &new_dim(conduit::DataType::int64(1), &new_dim_val, true);
+           new_dim.to_data_type(global_dim.dtype().id(), global_dim);
+       }
+
```

Comment:
Since this transforms the values in `global_origin`, it's no longer a global origin at the end of this, so the name is a bit confusing.  Perhaps it could be named `window_origin` with comments that indicate that it starts with global values and ends with local values.